src/index.ts

```diff
@@ -26,19 +26,18 @@ function createPerTypeTokenMap(
   }, {}) as Partial<Record<UnifiedTokenType, Record<string, string>>>;
 }
 
+export type Options = {
+  prefix?: string | undefined;
+  cssVarPrefix?: string | undefined;
+};
```

Comment:
```
export type Options = {
  prefix?: string;
  cssVarPrefix?: string;
};
```
The undefined definition maybe redundant?

Reply:
Actually it depends on user's tsconfig.
If user enables `exactOptionalPropertyTypes`, user cannot do `options.prefix = undefined` though it's allowed in our plugin.
see https://www.typescriptlang.org/tsconfig/#exactOptionalPropertyTypes.